projects/components/src/filtering/filter-button/filter-button.component.ts

```diff
@@ -77,4 +82,21 @@ export class FilterButtonComponent implements OnChanges {
       .lookup(attribute.type)
       .buildFiltersForSupportedOperators(attribute, value, subpath);
   }
+
+  private buildSimplifiedFilters(attribute: FilterAttribute, value?: FilterValue): Filter[] {
```

Comment:
This function can be renamed and remove the value arg. Renamed because it's "simplified" in that it's hardcoded to null (so maybe buildFiltersForNullValue), and remove the arg because we've already checked it outside before calling it. The behavior here is dependent on it being null, so just use the value literal (which will also help readability). If value were undefined for example, we still want to use null in the filter, not undefined (since server only works in null, not undefined).

Reply:
Done, I kept the read-only const but updated the rest